sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/implementation/VirtualMachineUpdateTests.java

```diff
@@ -40,22 +41,22 @@ protected void cleanUpResources() {
     public void testVirtualMachineUpdate() {
         final String vmname = "javavm1";
 
-        final String mySqlInstallScript = "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/4397e808d07df60ff3cdfd1ae40999f0130eb1b3/mysql-standalone-server-ubuntu/scripts/install_mysql_server_5.6.sh";
-        final String installCommand = "bash install_mysql_server_5.6.sh Abc.123x(";
+        final String mySqlInstallScript = "https://raw.githubusercontent.com/Azure/azure-sdk-for-java/0f875b569cb30b3cd4232bd20abb02452431b1ad/sdk/resourcemanager/azure-resourcemanager-compute/src/test/assets/install_mysql_server_5.7.sh";
+        final String installCommand = "bash install_mysql_server_5.7.sh " + password();
         List<String> fileUris = new ArrayList<>();
         fileUris.add(mySqlInstallScript);
 
         VirtualMachine vm = computeManager.virtualMachines()
             .define(vmname)
-            .withRegion(Region.US_EAST)
+            .withRegion(Region.US_WEST3)
```

Comment:
curious why do we change region?

Reply:
not enough resource...

US_EAST seems to be among the most heavily used region.